starters/express-typescript-typeorm-postgres/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    tsconfigRootDir : __dirname, 
+    sourceType: 'module',
+  },
+  plugins: ['@typescript-eslint/eslint-plugin'],
+  extends: [
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+  ],
+  root: true,
+  env: {
+    node: true,
+    jest: true,
+  },
+  ignorePatterns: ['.eslintrc.js'],
+  rules: {
+    '@typescript-eslint/interface-name-prefix': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
You   ...like `any`?

> `any` is the mind-killer.
> `any` is the little-death that brings total obliteration.
>                    - Frank Herbert's Dune-of-TypeScript

I mean, it's fine, but are we sure we want to just allow any anytime we want?

Reply:
@BigAB do you want warning or error?

Reply:
I'm actually pretty okay with leaving this `off`, but if it were just based on my personal preference I would go with a warning I guess.

I am curious how the others feel though.

Reply:
set it to error